server/doc.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+mongoose.promise = Promise;
+const Schema = mongoose.Schema;
+const date = require('date-and-time');
+
+const docSchema = new Schema({
+  name: {type: String, required = true},
```

Comment:
required:true, not required=true
Fix for all

Reply:
fixed syntax